services/datamanager/builtin/builtin.go

```diff
@@ -404,6 +406,9 @@ func (svc *builtIn) Update(ctx context.Context, cfg *config.Config) error {
 		return err
 	}
 
+	if cfg.DisableDirectoryConfiguration && svcConfig.CaptureDir != "" && svcConfig.CaptureDir != viamCaptureDotDir {
+		return errCaptureDirectoryConfigurationDisabled
+	}
 	svc.captureDir = svcConfig.CaptureDir
```

Comment:
While you're here, could you also only change svc.captureDir if it's nonempty? Will make sure to document this, but we should always default to viamCaptureDotDir if it hasn't been set in the config.

```
if svcConfig.CaptureDir != "" {
   svc.captureDir = svcConfig.CaptureDir
}
```

For additional context: we currently pass an absolute path (`/.viam/capture`) from the app frontend since we don't have a way of accessing os.getenv("HOME"), but I've already wanted to switch this over to just passing an empty string and allowing RDK to resolve unless overridden by the user, especially as we've run into issues like https://viam.atlassian.net/browse/DATA-954.